.github/workflows/CI.yml

```diff
@@ -74,8 +74,9 @@ jobs:
       with:
         path: |
           ~/.cache/pip
+          ~/.cache/srtm
           .tox
-        key: ${{ runner.os }}-tox-${{ matrix.python-version }}-${{ hashFiles('setup.cfg') }}
+        key: ${{ runner.os }}-tox-${{ matrix.python-version }}-${{ hashFiles('setup.cfg', '') }}
```

Comment:
why did you remove the `pyproject.toml` from the cache key?

Reply:
I did not mean to. Will reinstate.